src/routes/current-ticket/ticket-details-sidebar/TicketFileList.jsx

```diff
@@ -0,0 +1,53 @@
+import InsertDriveFileIcon from '@mui/icons-material/InsertDriveFile';
+import UploadFileIcon from '@mui/icons-material/UploadFile';
+import { Box, Paper, Typography } from '@mui/material';
+import { useState } from 'react';
+;
+
+export const TicketFileList = ({files}) => (
+  <Box display="flex" flexWrap="wrap" gap={2}>
+    <UploadFileButton onClick={console.log}/>
+    {
+      files.map(({title, link}) => (
+        <File key={link} title={title}/>
+      ))
+    }
+  </Box>
+);
+
+
+const File = ({title}) => {
+  const [isHovering, setIsHovering] = useState(false);
+  
+  return (
+    <Paper
+      sx={{display: 'flex', flexDirection: 'column', alignItems: 'center', p: 2, cursor: 'pointer'}}
+      elevation={isHovering ? 3 : undefined}
+      onMouseEnter={() => setIsHovering(true)}
+      onMouseLeave={() => setIsHovering(false)}
+    >
+      <InsertDriveFileIcon/>
+      <Typography sx={{direction: 'rtl'}}>{title}</Typography>
+    </Paper>
+  );
+}
+
+const UploadFileButton = ({onClick}) => {
+  const [isHovering, setIsHovering] = useState(false);
+
+  return (
+    <Paper
+      sx={{
+        display: 'flex', flexDirection: 'column', alignItems: 'center', p: 2, cursor: 'pointer',
+        color: theme => theme.palette.primary.light
+      }}
+      onClick={onClick}
+      elevation={isHovering ? 3 : undefined}
```

Comment:
null

Reply:
default value for elevation only triggers when undefined